honeycombio/honeycomb-opentelemetry-web
# chore: 🔧 add unit test setup

## Which problem is this PR solving?
- Getting the repo ready with basic testing and linting, will be closely followed up by a PR to connect this to CI to run on every PR and the main branch.
- Closes #15 

## Short description of the changes
- Setup jest testing for unit tests
- Add lint commands to package.json and fix lint issues

## How to verify that this has the expected result
- Run `npm run test` and you should see one passing test
- Run `npm run lint` and there should be no errors
